packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -239,7 +240,7 @@ class _NetworkProfilerControlsState extends State<_NetworkProfilerControls>
               _NetworkProfilerControls._includeTextWidth,
           onPressed: widget.controller.exportAsHarFile,
           gaScreen: gac.network,
-          gaSelection: gac.NetworkEvent.networkDownloadHar,
```

Comment:
this can stay in the analytics constants file. Sorry if that was confusing. All constants that we use for analytics should be in `shared/analytics/constants.dart` (or part of that file via a part file declaration like some of the other examples under `shared/analytics/constants/`). So if you move this particular enum back, you'll be able to access the constant here via `gac.NetworkEvents.downloadAsHar.name`.

Reply:
oh, alright, np moving it back